src/ListView/ListViewItem.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import {
+  ListViewExpand,
+  ListGroupItem,
+  ListGroupItemHeader,
+  ListGroupItemContainer
+} from './index'
+import ListViewRow from './ListViewRow'
+
+/**
+ * ListViewItem - main ListViewItem component which handles the expansion logic.
+ * ListViewItem is considered expandable if it has child props. In that case it
+ * renders ListGroupItemHeader and ListGroupItemContainer
+ */
+export default class ListViewItem extends React.Component {
+  constructor() {
+    super()
+    this.state = { expanded: false }
+  }
+
+  toggleExpanded(index) {
+    const { onExpand, onExpandClose } = this.props
+    if (this.state.expanded) {
+      onExpandClose && onExpandClose()
+    } else {
+      onExpand && onExpand()
+    }
+    this.setState(prevState => ({ expanded: !prevState.expanded }))
+  }
+
+  render() {
+    const {
+      children,
+      stacked,
+      actions,
+      additionalItems,
```

Comment:
It seems that this is called both `additionalItems` as well as `ListViewAdditionalInfo`.  I wonder if we should converge on one name to avoid confusion.

Reply:
Would changing the prop to additionalInfo be sufficient?